example/astx/lexer/lexer.go

```diff
@@ -111,16 +111,13 @@ func (l *Lexer) Reset() {
 	l.pos = 0
 }
 
-/*
-Lexer symbols:
-0: '_'
-1: '_'
-2: ' '
-3: '\t'
-4: '\n'
-5: '\r'
-6: 'a'-'z'
-7: 'A'-'Z'
-8: '0'-'9'
-9: .
-*/
+// Lexer symbols:// 0: '_'
```

Comment:
I think when comments are grouped and over multiple lines, it might be better to keep them grouped, such as in this case.